target/linux/bcm4908/base-files/lib/upgrade/platform.sh

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: GPL-2.0 OR BSD-2-Clause
 
-RAMFS_COPY_BIN="bcm4908img expr egrep fdtget fw_printenv fw_setenv tr"
+RAMFS_COPY_BIN="bcm4908img expr grep fdtget fw_printenv fw_setenv tr"
```

Comment:
Why don't you add a `-E` here?

Reply:
This is a list of binaries, not commands.